CafeApp/database/productsDB.tsx

```diff
@@ -0,0 +1,73 @@
+// This is our products database for now. Later we will replace this with database and redux state
+export const globoProducts =[
```

Comment:
space is needed after equals sign. I recomment installing prettier extension to visual code and set up it for automatic formating (or on save.)